### ISSUE TITLE
Readme file after "create"?

### ISSUE THREAD

**Issue by potherca** [WouterHendriks/wh-creator#5]
Although the `.git` repo is removed and a new git repository is initialised, the `README.md` file is not removed.

Either useful information should be added to the README or it should simply be removed by the create script.

**Comment by WouterHendriks** [comment]
Possible stuff to print, inspired by earlier modules:

```
# `name of site`

## URLs
Your test/live URLs here

## Backend
Your backend URLs here

## Installation
git clone URL-to-Git-repository "$(wh getdatadir)installedmodules/`modulename`"

## To satisfy the module dependencies:
- whcd `modulename`/webdesigns/`modulename`/
- if whcd is unavailable, try cd ~/projects/webhare/whtree/var/installedmodules/`modulename`/webdesigns/`modulename`/
- wh noderun npm install
- wh noderun bower install
```

More useful things can be added later I guess.

**Comment by potherca** [comment]
:+1:  

Also, if a question was added to the script ("Would you like to add a remote git repository Y/N"), that could be (re)used to put in the README file.

**Comment by WouterHendriks** [comment]
README is fixed in 88970df & ea527b2.

Question is a good idea, will add later.

**Comment by potherca** [comment]
Closing this ticket, as it is mostly resolved. "The question" can be added without having a ticket.
